Documentation/api-reference.md

```diff
@@ -0,0 +1,339 @@
+
+#### Java
+
+##### Syntax
+```java
+public static void getIdentities(final AdobeCallback<IdentityMap> callback);
+```
+* _callback_ is invoked after the identities are available. The return format is an instance of [IdentityMap](api-reference.md#identitymap). The callback may be invoked on a different thread.
+
+##### Example
+```java
+Identity.getIdentities(new AdobeCallback<IdentityMap>() {    
+    @Override    
+    public void call(IdentityMap identityMap) {        
+         //Handle the IdentityMap returned here    
+    }
+});
+```
+
+------
+
+### getUrlVariables
+> :information_source: This method was added in Edge Identity version 1.1.0.
+
+This API returns the identifiers in URL query parameter format for consumption in **hybrid mobile applications**. There is no leading & or ? punctuation as the caller is responsible for placing the variables in their resulting URL in the correct locations. If an error occurs while retrieving the URL variables, the callback handler will be called with a null value. Otherwise, the encoded string is returned, for example: `"adobe_mc=TS%3DTIMESTAMP_VALUE%7CMCMID%3DYOUR_ECID%7CMCORGID%3D9YOUR_EXPERIENCE_CLOUD_ID"`
+
+* The `adobe_mc` attribute is an URL encoded list that contains:
+  * `MCMID` - Experience Cloud ID \(ECID\)
+  * `MCORGID` - Experience Cloud Org ID
+  * `TS` - A timestamp taken when this request was made
+
+> :memo: When [AdobeCallbackWithError](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#public-classes) is provided, and you are fetching the url variables from the Mobile SDK, the timeout value is 500ms. If the operation times out or an unexpected error occurs, the `fail` method is called with the appropriate [AdobeError](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#public-classes).
+
+#### Java
+
+##### Syntax
+```java
+public static void getUrlVariables(final AdobeCallback<String> callback);
+```
+* _callback_ has an NSString value that contains the visitor identifiers as a query string after the service request is complete.
+
+##### Example
+```java
+Identity.getUrlVariables(new AdobeCallback<String>() {    
+    @Override    
+    public void call(String urlVariablesString) {        
+        //handle the URL query parameter string here
+        //For example, open the URL on the device browser        
+        //        
+        Intent i = new Intent(Intent.ACTION_VIEW);        
+        i.setData(Uri.parse("https://example.com?" + urlVariablesString));        
+        startActivity(i);    
```

Comment:
Can we update this example to use webView.loadUrl https://developer.android.com/reference/android/webkit/WebView#loadUrl(java.lang.String) ?